airflow/utils/helpers.py

```diff
@@ -22,9 +22,10 @@
 from functools import reduce
 from itertools import filterfalse, tee
 from typing import Any, Callable, Dict, Generator, Iterable, List, Optional, TypeVar
-from urllib import parse
 
+from flask import url_for
 from jinja2 import Template
+from urllib import parse
```

Comment:
This should be put in the block above, with other stdlib imports.

Reply:
oooh. I was wondering why there was a separation there.

Will do.